src/app/products/loan-products/common/loan-product-summary/loan-product-summary.component.ts

```diff
@@ -331,4 +339,8 @@ export class LoanProductSummaryComponent implements OnInit, OnChanges {
     return this.accounting.getAccountRuleName(value.toUpperCase());
   }
 
+  mapHumanReadableValueStringEnumOptionDataList(incomingParameter : StringEnumOptionData[]): string[] {
```

Comment:
Use the `OptionData` already defined in 'app/shared/models/option-data.model'

Reply:
Its not the same. OptionData id is number. StringEnumOptionData id is string